features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,426 @@
+<?php
+/**
+ * Behat tests.
+ *
+ * @package ee-cli
+ */
+
```

Comment:
Remove feature directory as it's copied from `site-wp-command`. You need to add tests but it might take some time so for now remove it, add it later